### ISSUE TITLE
How to send a message to the webmaster?

### ISSUE THREAD

**Issue by h4ck3rm1k3** [e14n/pump.io#1103]
We need a feature to send a message to the webmaster of a running site.

**Comment by larjona** [comment]
IMHO it's not needed a feature, just all the users should know who is handling their server, and how to contact them.

One possible approach:
1.- include a new variable in pump.io.json file with the contact email address
2.- modify public/template/welcome.utml (the welcome message) and public/template/main.utml (the main page of the site), to include some contact words and use the variable
3.- modify the server app to make it pass the variable to the templates.

Another approach could be that server admins modify public/template/main.utml  to hardcode the contact info, as did the @luisgf in his server https://mipump.es/

**Comment by h4ck3rm1k3** [comment]
We should make it easy for people to turn on. I am finding bugs and problems and need to report bugs to the owner. At least we should have a way to report bugs to the owner. All of these things can bug turned off of course.

**Comment by larjona** [comment]
I agree. In addition to what I posted, we are willing to maintain a list of pump nodes and maybe we could include contact information for each of them (if each node admin allows, of course).

We receive also bug reports here that should be reported to the corresponding sysadmin; maybe we should turn on a label? Not sure about that, because hopefully the senders get a quick reply with the contact info for their server (if somebody in the community knows him/her).

About this issue in particular, if you see that my first approach is do-able, do you mind to send a pull request? I could try but I'm not a coder, so it would be very hard for me.

Thanks for your interest in improving pump,io!

**Comment by h4ck3rm1k3** [comment]
yes I will work on these issues when I learn the code more, right now I am working on getting my server running
